openfl-tutorials/experimental/Workflow_FedProx_Keras_MNIST.ipynb

```diff
@@ -0,0 +1,1304 @@
+      "123/123 [==============================] - 1s 12ms/step - loss: 0.4056 - accuracy: 0.8797[9\u001b[9\u001b[9\u001b[9\u001b[9\u001b[9\u001b[9\n",
+      "\u001b[0m\u001b[94mTest set: Avg. loss: 0.4056294560432434, Accuracy: 87.97%\u001b[0m\u001b[94m\n",
+      "\u001b[0m\u001b[94mDoing local model validation for collaborator Bangalore: 0.8796722888946533\u001b[0m\u001b[94m\n",
+      "\u001b[0m\u001b[94mSaving data artifacts for local_model_validation\u001b[0m\u001b[94m\n",
+      "\u001b[0m\u001b[94mSaved data artifacts for local_model_validation\u001b[0m\u001b[94m\n",
+      "\u001b[0mShould transfer from local_model_validation to join\n",
+      "\n",
+      "Calling join\n",
+      "\u001b[94mAverage aggregated model validation values = 0.8868495523929596\u001b[0m\u001b[94m\n",
+      "\u001b[0m\u001b[94mAverage training loss = 0.34918831288814545\u001b[0m\u001b[94m\n",
+      "\u001b[0m\u001b[94mAverage local model validation values = 0.8157957792282104\u001b[0m\u001b[94m\n",
+      "\u001b[0m\u001b[94mTaking FedAvg of models of all collaborators\u001b[0m\u001b[94m\n",
+      "\u001b[0m\u001b[94mSaving data artifacts for join\u001b[0m\u001b[94m\n",
+      "\u001b[0m\u001b[94mSaved data artifacts for join\u001b[0m\u001b[94m\n",
+      "\u001b[0m\n",
+      "Calling internal_loop\n",
+      "\u001b[94mSaving data artifacts for internal_loop\u001b[0m\u001b[94m\n",
+      "\u001b[0m\u001b[94mSaved data artifacts for internal_loop\u001b[0m\u001b[94m\n",
+      "\u001b[0m\n",
+      "Calling end\n",
+      "\u001b[94mThis is the end of the flow\u001b[0m\u001b[94m\n",
+      "\u001b[0mSaving data artifacts for end\n",
+      "Saved data artifacts for end\n"
+     ]
+    }
+   ],
+   "source": [
+    "flflow.run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dce61095",
+   "metadata": {},
+   "source": [
+    "Now, let's train the model using the federated setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f278871e",
+   "metadata": {},
+   "source": [
+    "Now, after the training, let's evaluate the aggregated model accuracy again"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 83,
+   "id": "289215be",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "563/563 [==============================] - 7s 13ms/step - loss: 0.3596 - accuracy: 0.8944\n",
+      "Test set: Avg. loss: 0.35956209897994995, Accuracy: 89.44%\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "0.894444465637207"
+      ]
+     },
+     "execution_count": 83,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "inference(flflow.model, (test_images, test_labels), 32)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "a61a876d",
+   "metadata": {},
+   "source": [
+    "Now that the flow is complete, let's dig into some of the information captured along the way"
```

Comment:
Not sure the purpose of all cells after this comment. Suggest adding outputs, if relevant, or remove those cells altogether.